workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/TypedViewRegistry.kt

```diff
@@ -22,7 +22,7 @@ import kotlin.reflect.KClass
  * rendering types.
  */
 @WorkflowUiExperimentalApi
-internal class BindingViewRegistry private constructor(
+internal class TypedViewRegistry private constructor(
```

Comment:
@zach-klippenstein Heads up on this one, the only "non-mechanical" renaming. WDYT?

Reply:
`CoreViewRegistry`? `BaseViewRegistry`?

Reply:
Discussion below.

Reply:
Not seeing anything else about this.